### ISSUE TITLE
New terminal windows only open in a new tab

### ISSUE THREAD

**Issue by Pawper** [facebook/nuclide#1557]
### Issue and Steps to Reproduce
Opening a new terminal window within Atom IDE always opens in a new tab and this cannot be customized.

### Expected Behavior
An option within the Atom IDE package settings to open in a preferential area of the workspace. Personally I would prefer to open the terminal window in a collapsible dock below the workspace item the terminal window is launched for.

### Actual Behavior
The terminal opens in a tab that must then be manually moved to the preferential area of the workspace.

### Versions
* Atom: Latest
* Client OS: Windows 10

**Comment by pelmers** [comment]
An option for the preferred location is likely to come soon. @wbinnssmith

**Comment by wbinnssmith** [comment]
We shipped a fix in last week's release (338, I believe) that fixes this by making terminals behave like other Atom pane items. This way, if you drag a terminal around, future terminals will also open in that location.

We also changed the default location for the terminal to the bottom dock.

We also introduced a new command, `atom-ide-terminal:toggle`, to toggle the visibility of all terminals.

Thanks for reporting @Pawper!
